packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -498,7 +498,7 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
 
   bool get hasScrolledBody {
     for (_NestedScrollPosition position in _innerPositions) {
-      if (position.pixels > position.minScrollExtent)
+      if (position.minScrollExtent != null && position.pixels > position.minScrollExtent)
```

Comment:
This seems to fix my issue. I will proceed to use it as a workaround in the meantime